App.js

```diff
@@ -77,6 +83,35 @@ class App extends Component {
     LayoutAnimation.spring();
   }
 
+  getCurrentRouteName(navigationState) {
+    if (!navigationState) {
+      return null;
+    }
+    const route = navigationState.routes[navigationState.index];
+
+    if (route.routes) {
+      return this.getCurrentRouteName(route);
+    }
+
+    return route.routeName;
+  }
+
+  setStatusBar = ({ translucent, backgroundColor, barStyle }, routeName) => {
+    StatusBar.setTranslucent(translucent);
+    StatusBar.setBackgroundColor(backgroundColor(routeName));
+    StatusBar.setBarStyle(barStyle);
+  };
+
+  statusBarHandler(prev, next) {
+    const routeName = this.getCurrentRouteName(next);
+
+    const statusBarConfig = lightScreens.includes(routeName)
+      ? lightStatusBar
+      : darkStatusBar;
```

Comment:
@jouderianjr How about moving these lines into `src/config/status-bar.js` and only exporting 1 function called `getStatusBarConfig(routerName)`? Suppose the 3rd kind of status bar was introduced, only the config file should be updated.